### PR TITLE
Addresses 3 issues

### PR DESCRIPTION
**Pull request by Teoxoy**
https://github.com/material-components/material-components-web/issues/999 - Partially completed (I need answers)
https://github.com/material-components/material-components-web/issues/1018 - Added new feature to MDCTextfield
https://github.com/material-components/material-components-web/issues/1049 - Updated READMEs to fix es2015 imports
Sorry for referencing 3 issues in 1 PR.

### CODE REVIEW

File: packages/mdc-switch/mdc-switch.scss [hunk 1]
```diff
@@ -192,3 +193,11 @@
     }
   }
 }
+
+.mdc-switch-label {
```
Comment (lynnjepsen) [inline packages/mdc-switch/mdc-switch.scss]: Take this out of this PR. I want to discuss more if we want one `mdc-label` for switch, checkbox, and radio...or if we need three classes.

File: packages/mdc-textfield/README.md [hunk 2]
```diff
@@ -153,9 +153,9 @@ by HTML5's form validation API.
 </div>
 ```
 
-An input's validity is checked via `checkValidity()` on blur, and the styles are updated
-accordingly. When using the `required` attribute, an asterisk will be automatically appended to the
-label text, as per the spec.
+By default an input's validity is checked via `checkValidity()` on blur, and the styles are updated
```
Comment (lynnjepsen) [inline packages/mdc-textfield/README.md]: Move this Textfield.valid out to another PR

File: packages/mdc-textfield/README.md [hunk 3]
```diff
@@ -324,6 +324,10 @@ with the corresponding id within the document and automatically assign it to thi
 Boolean. Proxies to the foundation's `isDisabled/setDisabled` methods when retrieved/set
 respectively.
 
+##### MDCTextfield.valid
```
Comment (lynnjepsen) [inline packages/mdc-textfield/README.md]: Move out to Textfield.valid PR

File: packages/mdc-textfield/foundation.js [hunk 4]
```diff
@@ -57,6 +57,7 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.inputBlurHandler_ = () => this.deactivateFocus_();
     this.inputInputHandler_ = () => this.autoCompleteFocus_();
     this.inputKeydownHandler_ = () => this.receivedUserInput_ = true;
+    this.useNativeValidityChecking_ = true;
```
Comment (lynnjepsen) [inline packages/mdc-textfield/foundation.js]: Move this entire file out to Textfield.valid PR

**Comment by Teoxoy** [comment]
I split the 3 issues across 3 branches in my fork and i closed this one.

**Pull request closed without merging**